src/main/kotlin/uk/gov/dluhc/printapi/rest/aed/AnonymousElectorDocumentController.kt

```diff
@@ -163,3 +183,9 @@ class QueryStringParameterPropertyEditor : PropertyEditorSupport() {
             ?.let { it.ifEmpty { null } }
     }
 }
+
+private fun UpdateAnonymousElectorDocumentRequest.validate() {
+    if (email.isNullOrBlank() && phoneNumber.isNullOrBlank()) {
+        throw UpdateAnonymousElectorDocumentValidationException("At least one editable field must be provided")
```

Comment:
`Either email or phoneNumber must be provided` because this check is mostly around these fields

Reply:
I had that originally, but changed it because someone could add another field, but forget to update this message 😉

Reply:
Fantastic 👍🏼

Reply:
Great stuff.